tests/functional/tests.py

```diff
@@ -2,7 +2,7 @@
 import os
 
 from subprocess import call
-from raven.utils.compat import TestCase
+from raven.utils.testutils import BaseTestCase as TestCase
```

Comment:
The renaming isn't an issue to me. However, to import this we'll need to have exam installed which may not be what I'm looking for.
Anyway I can always refactor this later.